src/services/codefixes/fixClassDoesntImplementInheritedAbstractMember.ts

```diff
@@ -1,6 +1,7 @@
 /* @internal */
 namespace ts.codefix {
     const errorCodes = [
+        Diagnostics.Non_abstract_class_0_does_not_implement_all_abstract_members_of_1.code,
         Diagnostics.Non_abstract_class_0_does_not_implement_inherited_abstract_member_1_from_class_2.code,
         Diagnostics.Non_abstract_class_expression_does_not_implement_inherited_abstract_member_0_from_class_1.code,

```

Comment:
Are these needed anymore now that they're related spans? Is it even possible to fix just one missing member in the first place?

Reply:
Oh, of course. The related spans are not really related spans, just elaborations. So these are definitely not needed.